javascript-and-json/main.js

```diff
@@ -0,0 +1,26 @@
+var books = [
+  {
+    title: 'Harry Potter',
+    isbn: 23523453425
+  },
+  {
+    title: 1984,
+    isbn: 98762345
+  },
+  {
+    title: 'The 1619 Project',
+    isbn: 1234089
+  }
+];
+console.log('type of ', typeof books); // starts off as js object
+console.log('books', books);
+
+console.log('json stringify ', JSON.stringify(books)); // is serialized into a json string
+console.log('type of json stringify ', typeof JSON.stringify(books));
+
+var student = JSON.stringify({ name: 'julius', NumberId: 2345 });// starst as a JSON string
```

Comment:
you don't want to use `stringify` here, you want to manually create a JSON string.